Library/Formula/groonga.rb

```diff
@@ -18,9 +20,9 @@ class Groonga < Formula
   depends_on "lz4" => :optional
   depends_on "openssl"
 
-  depends_on "glib" if build.include? "enable-benchmark"
+  depends_on "glib" if build.with? "benchmark"
 
-  option "enable-benchmark", "Enable benchmark program for developer use"
+  deprecated_option "enable-banchmark" => "with-benchmark"
```

Comment:
There's a tiny typo here, `enable-banchmark` instead of `enable-benchmark`.

Otherwise :+1: :heart: